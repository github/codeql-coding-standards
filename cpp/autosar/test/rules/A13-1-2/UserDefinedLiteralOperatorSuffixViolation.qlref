rules/A13-1-2/UserDefinedLiteralOperatorSuffixViolation.ql