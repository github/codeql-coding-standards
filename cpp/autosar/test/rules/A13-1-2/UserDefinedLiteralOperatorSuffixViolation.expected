| test.cpp:1:6:1:17 | operator ""x | User defined literal operator should start with an underscore followed by one or more letters |
| test.cpp:2:6:2:17 | operator ""X | User defined literal operator should start with an underscore followed by one or more letters |
| test.cpp:3:6:3:17 | operator ""_ | User defined literal operator should start with an underscore followed by one or more letters |
| test.cpp:4:6:4:18 | operator ""_1 | User defined literal operator should start with an underscore followed by one or more letters |
| test.cpp:5:6:5:19 | operator ""_x1 | User defined literal operator should start with an underscore followed by one or more letters |
